app/lib/methods/sendFileMessage.ts

```diff
@@ -85,6 +85,13 @@ export function sendFileMessage(
 				});
 			}
 
+			if (fileInfo.msg) {
```

Comment:
Should it really be different from the current behavior?
We already have the option to add a message to an image on ShareView.
What's going on here?

Reply:
the image's description already exists and is passed to the server by `description`, and now we are implementing the ability to send the quote together, and the server receives the quotes by this param `msg`